internal/server/stat/golden/get_stat_set_series/IG_nyt_covid_cases.json

```diff
@@ -0,0 +1,2228 @@
+            "2021-11-06": 147918,
+            "2021-11-07": 147918,
+            "2021-11-08": 148453,
+            "2021-11-09": 148582,
+            "2021-11-10": 148680,
+            "2021-11-11": 148888,
+            "2021-11-12": 148888,
+            "2021-11-13": 149251,
+            "2021-11-14": 149251,
+            "2021-11-15": 149582,
+            "2021-11-16": 149743,
+            "2021-11-17": 149882,
+            "2021-11-18": 150095,
+            "2021-11-19": 150298,
+            "2021-11-20": 150298,
+            "2021-11-21": 150298,
+            "2021-11-22": 150856,
+            "2021-11-23": 151010,
+            "2021-11-24": 151169,
+            "2021-11-25": 151169,
+            "2021-11-26": 151169,
+            "2021-11-27": 151169,
+            "2021-11-28": 151169,
+            "2021-11-29": 151905,
+            "2021-11-30": 152089,
+            "2021-12-01": 152268,
+            "2021-12-02": 152543,
+            "2021-12-03": 152857,
+            "2021-12-04": 152857,
+            "2021-12-05": 152857,
+            "2021-12-06": 153696,
+            "2021-12-07": 153892,
+            "2021-12-08": 154047,
+            "2021-12-09": 154364,
+            "2021-12-10": 154536,
+            "2021-12-11": 154536,
+            "2021-12-12": 154536,
+            "2021-12-13": 155142,
+            "2021-12-14": 155362,
+            "2021-12-15": 155504,
+            "2021-12-16": 155716,
+            "2021-12-17": 156031,
+            "2021-12-18": 156031,
+            "2021-12-19": 156031,
+            "2021-12-20": 156837,
+            "2021-12-21": 157215,
+            "2021-12-22": 157557,
+            "2021-12-23": 158113,
+            "2021-12-24": 158727,
+            "2021-12-25": 158727,
+            "2021-12-26": 158727,
+            "2021-12-27": 159974,
+            "2021-12-28": 161759,
+            "2021-12-29": 162666,
+            "2021-12-30": 164698,
+            "2021-12-31": 166609,
+            "2022-01-01": 166609,
+            "2022-01-02": 166609,
+            "2022-01-03": 174330,
+            "2022-01-04": 176928,
+            "2022-01-05": 178916,
+            "2022-01-06": 183551,
+            "2022-01-07": 188711,
+            "2022-01-08": 188711,
+            "2022-01-09": 188711,
+            "2022-01-10": 203430,
+            "2022-01-11": 208256,
+            "2022-01-12": 212940,
+            "2022-01-13": 218660,
+            "2022-01-14": 224378,
+            "2022-01-15": 224378,
+            "2022-01-16": 224378,
+            "2022-01-17": 239685,
+            "2022-01-18": 243326,
+            "2022-01-19": 245673,
+            "2022-01-20": 253711,
+            "2022-01-21": 257994,
+            "2022-01-22": 257994
+          },
+          "metadata": {
+            "importName": "NYT_COVID19",
+            "provenanceUrl": "https://www.nytimes.com/article/coronavirus-county-data-us.html",
+            "measurementMethod": "NYT_COVID19_GitHub"
+          }
```

Comment:
So this one works!

Reply:
Yep! Need to add "infrequent" and see for more.